botocore/args.py

```diff
@@ -59,6 +59,10 @@
 # Maximum allowed length of the ``user_agent_appid`` config field. Longer
 # values result in a warning-level log message.
 USERAGENT_APPID_MAXLEN = 50
+# Minimum and maximum allowed value of the `request_min_compression_size_bytes`
+# config field. Lower or higher values result in an InvalidConfigError raised.
```

Comment:
There's a question here of if these should be at module scope. That probably also should have been caught for `USERAGENT_APPID_MAXLEN`. They're now importable and not explicitly private. If they're only used in the single function can we just define them to that scope? We can always hoist higher later if needed, but this may be premature.

Reply:
Yeah that's totally fair. There's no need to expose these at all. I'll remove the comments as well since the uses should be more inherently obvious when defined in the function.